bskyweb/cmd/bskyweb/server.go

```diff
@@ -221,6 +221,10 @@ func serve(cctx *cli.Context) error {
 	e.GET("/profile/:handleOrDID/post/:rkey/liked-by", server.WebGeneric)
 	e.GET("/profile/:handleOrDID/post/:rkey/reposted-by", server.WebGeneric)
 
+    // starter packs
+	e.GET("/starter-pack/:handleOrDID/:rkey", server.WebGeneric)
```

Comment:
So for signed in users we use `starter-pack` and for signed out ones its `start`. That's because the App Clip is going to try and present itself on a specific URL path.